cmd/containerd-shim-runhcs-v1/options/runhcs.proto

```diff
@@ -4,6 +4,7 @@ package containerd.runhcs.v1;
 
 import weak "gogoproto/gogo.proto";
 import "google/protobuf/timestamp.proto";
+import "github.com/containerd/cgroups/metrics.proto";
```

Comment:
It is a bit weird to still call this directory "options", probably rename it to types or api?

Reply:
That's a good idea. I think I will add a dir called "stats" and then put the stats proto in it.